features/testdata/es_multiple_search_dataset_results.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
Might be one for later, but if these testdatas contain the same data in topics it might be worth just renaming and getting them together rather than multiple duplications